examples/passing/PolyLabels.purs

```diff
@@ -0,0 +1,65 @@
+module Main where
+
+import Prelude
+import Control.Monad.Eff
+import Control.Monad.Eff.Console
+import Data.Symbol (class IsSymbol, SProxy(..), reflectSymbol)
+
+foreign import unsafeGet
+  :: forall r a
+   . String
+  -> Record r
+  -> a
+
+foreign import unsafeSet
+  :: forall r1 r2 a
+   . String
+  -> a
+  -> Record r1
+  -> Record r2
+
+get
+  :: forall r l a
+   . IsSymbol l
+  => Lookup l r a
+  => SProxy l
+  -> Record r
+  -> a
+get l = unsafeGet (reflectSymbol l)
+
+set
```

Comment:
If we could construct a singleton row from a symbol and a type I think we could do this.  Something like this?
```purescript
class Singleton (l :: Symbol) (t :: Type) (r :: # Type) | l t -> r

set :: forall l a b u r1 r2.
  IsSymbol l =>       -- l = "x"
  Singleton l b u =>  -- l = "x", b = X, u = (x :: X)
  Union u r1 r2 =>    -- u = (x :: X), r1 = (x :: Y, z :: Z), r2 = (x :: X, x :: Y, z :: Z)
  Lookup l r1 a =>    -- l = "x", r1 = (x :: Y, z :: Z), a = Y
  SProxy l -> b -> Record r1 -> Record r2
```